feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseSample.java

```diff
@@ -0,0 +1,423 @@
+      return timestamp.get();
+    }
+  }
+
+  /**
+   * Creates UI with two buttons and a label component.
+   */
+  private void setUpControlsVBox() {
+
+    // create label to show number of added features
+    label = new Label("Number of features added: ");
+    label.setStyle("-fx-text-fill: white");
+
+    // create button to show alert and close geodatabase
+    createGeodatabaseButton = new Button("Create Mobile Geodatabase");
+    createGeodatabaseButton.setAlignment(Pos.CENTER);
+    createGeodatabaseButton.setDisable(true);
+
+    // create button to show features stored in the geodatabase
+    viewTableButton = new Button("View Table");
+    viewTableButton.setAlignment(Pos.CENTER);
+    viewTableButton.setDisable(true);
+
+    // create and configure a VBox
+    vBoxControls = new VBox(10);
+    vBoxControls.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.7)"),
+      CornerRadii.EMPTY, Insets.EMPTY)));
+    vBoxControls.setPadding(new Insets(10.0));
+    vBoxControls.setMaxSize(185, 100);
+    vBoxControls.setDisable(false);
+
+    // add the label and button to the VBos
+    vBoxControls.getChildren().addAll(label, createGeodatabaseButton, viewTableButton);
+  }
+
+  /**
+   * Create and load a new geodatabase file and feature layer from table description fields
+   */
+  private void createGeodatabase() throws IOException {
+
+    // create a directory for the geodatabase file
+    Path directoryPath = Paths.get(System.getProperty("user.dir"), "CreateMobileGeodatabase");
+    if(!Files.exists(directoryPath))
+      Files.createDirectories(directoryPath);
+
+    // create the path for the geodatabase file and delete duplicate file
+    geodatabasePath = Paths.get(directoryPath + "/LocationHistory.geodatabase");
+    // delete previous file
+    Files.deleteIfExists(geodatabasePath);
+
+    // create geodatabase from the specified mobile geodatabase file path
+    var geodatabaseFuture = Geodatabase.createAsync(geodatabasePath.toString());
+    geodatabaseFuture.addDoneListener(()-> {
+      try {
+        // get the instance of the mobile geodatabase
+        geodatabase = geodatabaseFuture.get();
+      } catch (InterruptedException | ExecutionException e) {
+      throw new RuntimeException(e);
+    }
+
+      // create a table description to store features as map points and set non-required properties to false
+      var tableDescription = new TableDescription("LocationHistory", SpatialReferences.getWgs84(), GeometryType.POINT);
+      tableDescription.setHasAttachments(false);
+      tableDescription.setHasM(false);
+      tableDescription.setHasZ(false);
+
+      // Set up the fields for the table:
+      // FieldType.OID is the primary key of the SQLite table.
+      // FieldType.Date is a date column used to store a Calendar date.
+      // FieldDescriptions can be a SHORT, INTEGER, GUID, FLOAT, DOUBLE, DATE, TEXT, OID, GLOBAL ID, BLOB, GEOMETRY, RASTER, or XML.
```

Comment:
this level of documentation is a bit heavy for what we usually include in samples (and in the sample we don't make use of the FieldType.Date): a lot of this will be found in the website documentation. we can keep this a little lighter and highlight the parts that might not be instantly obvious and are helpful for readability, e.g. 

```suggestion
      // set up the fields for the table, FieldType.OID is the primary key of the SQLite table
```
the user can easily see what field types are available without having to explicitly list them